R/pkg/inst/profile/shell.R

```diff
@@ -33,12 +33,12 @@
   sc <- SparkR:::callJStatic("org.apache.spark.sql.api.r.SQLUtils", "getJavaSparkContext", spark)
   assign("sc", sc, envir = .GlobalEnv)
   sparkVer <- SparkR:::callJMethod(sc, "version")
-  cat("\n Welcome to")
+  cat("\nWelcome to")
   cat("\n")
-  cat("    ____              __", "\n")
-  cat("   / __/__  ___ _____/ /__", "\n")
-  cat("  _\\ \\/ _ \\/ _ `/ __/  '_/", "\n")
-  cat(" /___/ .__/\\_,_/_/ /_/\\_\\")
+  cat("   ____              __", "\n")
+  cat("  / __/__  ___ _____/ /__", "\n")
+  cat(" _\\ \\/ _ \\/ _ `/ __/  '_/", "\n")
+  cat("/___/ .__/\\_,_/_/ /_/\\_\\")
```

Comment:
Don't these lines actually have to have 3 spaces in front to be consistent?

Reply:
Right, I was thinking of getting rid of one empty space per line. But I just reset these lines back to origin.

Reply:
Sorry I misunderstood it. Setting indent of these lines to 3 also.